### DISCUSSION TITLE
Connect link on an element with extern text

### DISCUSSION THREAD

**Discussion by malomiquel** [clientIO/joint#1860] [Q&A]
Hello,

I have a problem with the connection link magnet. I want to connect a link directly on the element but the text block the connection on element. There is a blank between my element and the link.
How can I solve this ?

Here is a screenshot of a connection.

<img width="225" alt="Capture d’écran 2022-10-25 125936" src="https://user-images.githubusercontent.com/83480447/197756615-c953d081-8e4e-4580-a86a-bf8292bbb066.png">
<img width="285" alt="Capture d’écran 2022-10-25 125906" src="https://user-images.githubusercontent.com/83480447/197756623-6cd4d6d8-e719-41e4-a899-0170ae0c5937.png">

**Comment by jamesgeorgewilliams** [comment]
Maybe you can take advantage of the `magnetSelector` attribute seen in the docs. [magnetSelector](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.attributes.magnetSelector) designates a sub-element as the target for the cell's magnet.

**Reply by kumilingus** [reply]
Here's an example: https://jsfiddle.net/kumilingus/7nhwxL03/
